### PR TITLE
Eip 1337 updated, pull request to wrong repo, please close

### PR DESCRIPTION
**Pull request by androolloyd**
~~EIP 1337 standard updated, added a new argument, a dynamic bytes array to the arguments for getSubscriptionHash and for execSubscription~~
please close this, pulled to the wrong repo.

### CODE REVIEW

File: EIPS/eip-1337.md [hunk 1]
```diff
@@ -0,0 +1,261 @@
+---
+eip: 1337
+title: [WIP] Subscriptions on the blockchain.
+author: Kevin Owocki <kevin@gitcoin.co> , Andrew Redden <andrew@blockcrushr.com>, Scott Burke <scott@blockcrushr.com> , Kevin Seagraves <k.s.seagraves@gmail.com> , Luka Kacil <luka.kacil@gmail.com>, Štefan Šimec <stefan.simec@gmail.com>, Piotr Kosiński (@kosecki123), ankit raj <tradeninja7@gmail.com>, John Griffin <john@atchai.com> , Nathan Creswell <nathantr@gmail.com>
+
```
Comment (axic) [inline EIPS/eip-1337.md]: I think this empty line will mess with automatic parsers.

File: EIPS/eip-1337.md [hunk 2]
```diff
@@ -0,0 +1,261 @@
...
+discussions-to: https://github.com/EthereumOpenSubscriptions/standard/issues or https://gitcoin.co/slack in #proj-subscriptions channel
```
Comment (axic) [inline EIPS/eip-1337.md]: I am not sure it is allowed to have multiple links here.

File: EIPS/eip-1337.md [hunk 3]
```diff
@@ -0,0 +1,261 @@
...
+type: Standards
+category: Interface
+created: 2018-08-01
+requires: ERC-20, ERC-948, EIP-165
```
Comment (axic) [inline EIPS/eip-1337.md]: I think this needs just numbers

File: EIPS/eip-1337.md [hunk 4]
```diff
@@ -0,0 +1,261 @@
...
+<!--The motivation is critical for EIPs that want to change the Ethereum protocol. It should clearly explain why the existing protocol specification is inadequate to address the problem that the EIP solves. EIP submissions without sufficient motivation may be rejected outright.-->
+Recurring payments are the bedrock of SaSS and countless other businesses, a robust specification for defining this interaction will enable a broad spectrum of revenue generation and business models.
+
+## Specification
+<!--The technical specification should describe the syntax and semantics of any new feature. The specification should be detailed enough to allow competing, interoperable implementations for any of the current Ethereum platforms (go-ethereum, parity, cpp-ethereum, ethereumj, ethereumjs, and [others](https://github.com/ethereum/wiki/wiki/Clients)).-->
+#### Enum Contract
+
+EIP-1337 Contracts should be compiled with a contract that references all the enumerations that are required for operation
+
+```SOLIDITY
```
Comment (axic) [inline EIPS/eip-1337.md]: I don't think this works with uppercase letters.

**Comment by Arachnid** [comment]
Please remove the IDE files from the PR, too.

**Pull request closed without merging**